lib/utils/Logger.js

```diff
@@ -163,17 +163,12 @@ Logger.prototype.data = function(data) {
  */
 Logger.prototype.result = function (result) {
 
-    // if result is undefined, null or empty don't print it
-    if(typeof result === 'undefined' || (typeof result === 'object' && !result) || (typeof result === 'string' && result.length === 0)) {
-        return;
```

Comment:
I am certainly willing to do this, but I find it less valuable. 

Did it return "", or did it return nothing? 

Did it return "(empty)" or did it return nothing which is being presented as "(empty)"?

I find that when I see null, I don't think, maybe it returned "null"

Just my 2 cents - you tell me what you prefer - you've earned it :)